package in.com.exceptionhandling;

public class TestTryCatch {
	
	public static void main(String[] args) {
		try {
			
			int a = 10;
			
			int b = 0;
			
			int c = a/b;
			
			System.out.println("Divison =" + c);
			
		} cach (Exception)
	}

}
